com.innstereo.innstereo.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "com.innstereo.innstereo",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
```

Comment:
3.30 is out since a while, have you tried it? It also includes gfortran if I'm not mistaken.

Reply:
I tried around but was not successful. Not sure how set the configs for the included gfortran.